pinot-tools/src/main/java/org/apache/pinot/tools/perf/QueryRunner.java

```diff
@@ -53,56 +52,58 @@ public class QueryRunner extends AbstractBaseCommand implements Command {
   private static final long NANO_DELTA = (long) 5E5;
   private static final String CLIENT_TIME_STATISTICS = "CLIENT TIME STATISTICS";
 
-  @CommandLine.Option(names = {"-mode"}, required = true,
-      description = "Mode of query runner (singleThread|multiThreads|targetQPS|increasingQPS).")
+  @CommandLine.Option(names = {"-mode"}, required = true, description = "Mode of query runner "
+      + "(singleThread|multiThreads|targetQPS|increasingQPS).")
```

Comment:
These changes do not seem to improve readability. IIRC we have disabled these kind of auto-reformat in the `Pinot Style`, can you try re-importing the style sheet?

Reply:
just did, no changes

Reply:
The new one will honor the new lines and not try to merge separate lines, so if you revert and reformat, it won't apply the same change here. Not important though :-P

Reply:
ah i see! will keep in mind for next time